common/api/imodeljs-backend.api.md

```diff
@@ -882,28 +897,21 @@ export class ConcurrencyControl {
     onSavedChanges(): void;
     // @internal (undocumented)
     onUndoRedo(): void;
-    // (undocumented)
-    openOrCreateCache(requestContext: AuthorizedClientRequestContext): Promise<void>;

```

Comment:
Did you mean to remove this method?

Reply:
I made it private, so it disappeared from the public API. I know that's a no-no, but it was undocumented, and it was a big mistake that it leaked out in to the public API in the first place. I'd rather break an app than leave it there.

Reply:
ok